StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -1441,9 +1451,50 @@ Scene* AndresTestScene(SceneManager* sm)
 
 
     /* ---------------------- Enemy -------------------------------- */
-    // Enemyfactory used in the wrong way. 
-    //EnemyFactory enH(scene);
-    //entity = enH.AddEnemy("enemy", enemyModel, 1000, float3{ 0, 10, 20 }, L"Bruh", F_COMP_FLAGS::OBB | F_COMP_FLAGS::CAPSULE_COLLISION, 0, 0.5, float3{ 0, 0, 0 }, "player", 25.0f, 7.0f, 0.5f, 10.0f);
+#pragma region enemyRangeTest
+    Entity* ent = scene->AddEntity("enemyRangeTest");
```

Comment:
I assume this functionality will later be added to the enemy factory?

Reply:
Correct.